jaeger-core/src/test/java/com/uber/jaeger/reporters/RemoteReporterTest.java

```diff
@@ -62,10 +62,11 @@ public void setUp() throws Exception {
   public void testRemoteReporterReport() throws Exception {
     Span span = (Span) tracer.buildSpan("raza").start();
     reporter.report(span);
-    Thread.sleep(5);
+    // do sleep until automatic flush happens on 'reporter'
+    Thread.sleep(flushInterval + 20); // added 20ms on top of 'flushInterval' to avoid corner cases
```

Comment:
in Go we just write a loop that checks & breaks or sleeps. Then after the loop we check again via assert.